TWLight/users/tests.py

```diff
@@ -1010,28 +1010,12 @@ def test_is_user_valid(self):
         )
         self.assertTrue(valid)
 
-    def test_editor_eligibility_functions(self):
```

Comment:
@jsnshrmn I removed this test function because I think we already test `editor_bundle_eligible` throughout different tests, including the one that I added in this PR. Let me know if you think I should refactor these tests instead of deleting them

Reply:
Yep, we have some redundancy there. The deletions are a-okay. We can just leave it at that since we have a big test redesign in our future.